app/javascript/components/KeyWordSearch.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import Button from 'react-bootstrap/lib/Button';
+import InputGroup from 'react-bootstrap/lib/InputGroup';
+import Form from 'react-bootstrap/lib/Form';
+import { faSearch } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+
```

Comment:
I think the more traditional capitalization is just 'Keyword' and not KeyWord'

Reply:
This has been taken care of in the new PR